Library/Formula/passenger.rb

```diff
@@ -23,6 +23,11 @@ def install
     rake "nginx"
     rake "webhelper"
 
+    # Fixes https://github.com/phusion/passenger/issues/1288
+    rm_rf "buildout/libev"
+    rm_rf "buildout/libeio"
+    rm_rf "buildout/cache"
```

Comment:
Well, the problem is as follows. The Passenger build system builds binaries, but due to the use of autoconf in libev and libeio, the temporary source path that Homebrew generates (/tmp/passenger-xxxx) is hardcoded into several files. This hardcoded path then causes problems later on.

This issue is specific to the Homebrew install path of Passenger, and doesn't appear anywhere else. This is why I believe this code belongs in the recipe, not in the Passenger Rakefile.